src/cgame/default/cg_main.c

```diff
@@ -106,6 +107,8 @@ static void Cg_Init(void) {
 
 	cg_fov = cgi.Cvar("cg_fov", "110.0", CVAR_ARCHIVE, NULL);
 	cg_fov_zoom = cgi.Cvar("cg_fov_zoom", "55.0", CVAR_ARCHIVE, NULL);
+	cg_fov_interpolate = cgi.Cvar("cg_fov_interpolate", "100", CVAR_ARCHIVE,
+			"Speed to change the fov at in x100 degrees/second. 0 disables, 1 is 100 deg/sec, 0.5 is 50 deg/sec");
```

Comment:
Not to make it more complicated, but why not just express it as the amount of time it takes to complete the zoom in? Slightly easier for users/editing I think, (especially since 1.0 = 100 deg/sec is a extremely weird mapping).